extensions/amp-analytics/0.1/vendors.js

```diff
@@ -1422,13 +1422,22 @@ export const ANALYTICS_CONFIG = /** @type {!JsonObject} */ ({
         'ampid=${clientId(_parsely_visitor)}',
       'pageview': '${basePrefix}&action=pageview',
       'heartbeat': '${basePrefix}&action=heartbeat' +
-      '&tt=${totalEngagedTime}&inc=${incrementalEngagedTime(parsely-js)}',
+      '&tt=${totalEngagedTime}&inc=${incrementalEngagedTime(parsely-js,true)}',
```

Comment:
nit: Please make this change in separate PR, thanks.

Reply:
Good call 👍